openshift/Dockerfile

```diff
@@ -4,8 +4,11 @@ ENV GO111MODULE=on \
     GOFLAGS=-mod=vendor
 
 COPY . /go/src/github.com/openshift/ansible-operator-plugins
-RUN cd /go/src/github.com/openshift/ansible-operator-plugins \
- && make build
+RUN cd /go/src/github.com/openshift/ansible-operator-plugins/openshift \
+ && export SIMPLE_VERSION="$(go run version.go)-ocp" \
+ && export GIT_VERSION="$(go run version.go)-ocp" \
+ && cd /go/src/github.com/openshift/ansible-operator-plugins \
+ && make -e build
```

Comment:
`-e` allows using the already set values of the environment variables, rather than the values which are set in the Makefile.